### PR TITLE
get marker genes with presto::top_markers

### PR DESCRIPTION
**Pull request by gerbeldo**
# Background

Filters marker genes using a couple more parameters, getting cleaner results, essentially.

#### Link to issue 
https://biomage.atlassian.net/browse/BIOMAGE-1578

#### Link to staging deployment URL 

https://ui-german-worker216-2.scp-staging.biomage.net/data-management

#### Anything else the reviewers should know about the changes here

# Changes
### Code changes

# Definition of DONE
Your changes will be ready for merging after each of the steps below have been completed:

### Testing
- [ ] Unit tests written
- [X] Tested locally with Inframock (with latest production data downloaded with biomage experiment pull)
- [X] Deployed to staging
- [x] Passed integration tests 

### Documentation updates
Is all relevant documentation updated to reflect the proposed changes in this PR?

- [ ] Relevant Github READMEs updated
- [ ] Relevant wiki pages created/updated

### Approvers
- [ ] Approved by a member of the core engineering team
- [x] (UX changes) Approved by vickymorrison (this is her username, tag her if you need approval)

### Just before merging:
- [x] After the PR is approved, the `unstage` script in here: https://github.com/biomage-ltd/biomage-utils is executed. This script cleans up your deployment to staging

### Optional
- [X] Photo of a cute animal attached to this PR

### CODE REVIEW

**Review by vickymorrison — APPROVED** [review]

File: r/R/worker_helpers.R [hunk 1]
```diff
@@ -21,18 +21,24 @@ getTopMarkerGenes <- function(nFeatures, data, cellSets) {
...
+      auc >= aucMin &
+      pct_in >= pctInMin &
+      pct_out <= pctOutMax) %>%
+    dplyr::group_by(feature) %>%
+    dplyr::slice(which.min(pval)) %>%
+    ungroup()
+
+  top_markers <- filtered_markers %>%
+    dplyr::arrange(group, desc(logFC)) %>%
+    dplyr::group_by(group) %>%
```
Comment (alexvpickering) [inline r/R/worker_helpers.R]:
```suggestion
    dplyr::slice(which.min(pval)) 
    
  top_markers <- filtered_markers %>%
    dplyr::group_by(group) %>%
    dplyr::arrange(desc(logFC)) %>%
```

File: r/R/worker_helpers.R [hunk 2]
```diff
@@ -21,18 +21,24 @@ getTopMarkerGenes <- function(nFeatures, data, cellSets) {
...
     dplyr::slice_head(n = nFeatures) %>%
-    arrange(group)
+    dplyr::ungroup()
```
Comment (alexvpickering) [inline r/R/worker_helpers.R]:
```suggestion
    arrange(group)
```

File: r/R/worker_helpers.R (hunk 1 above)
Comment (alexvpickering) [reply r/R/worker_helpers.R]: yours should work fine too just a few less lines

**Review by alexvpickering — APPROVED** [review]
consider changes

**Pull request merged**